packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -495,6 +530,15 @@ class AndroidWebViewController extends PlatformWebViewController {
     _onGeolocationPermissionsShowPrompt = onShowPrompt;
     _onGeolocationPermissionsHidePrompt = onHidePrompt;
   }
+
+  /// Sets a callback that notifies the host application of any log messages
+  /// written to the JavaScript console.
+  @override
+  Future<void> setConsoleLogCallback(
+      void Function(JavaScriptLogLevel level, String message)
+          onConsoleMessage) async {
+    _onConsoleLogCallback = onConsoleMessage;
```

Comment:
Should this also call `webChromeClient.setSynchronousReturnValueForOnConsoleMessage(true)` since the user is now handling the callback?

Reply:
You are correct, added.